src/governance/governance.cpp

```diff
@@ -1021,8 +1019,6 @@ bool CGovernanceManager::MasternodeRateCheck(const CGovernanceObject& govobj, bo
 
 bool CGovernanceManager::ProcessVote(CNode* pfrom, const CGovernanceVote& vote, CGovernanceException& exception, CConnman& connman)
 {
-    // TODO: drop cs_main here when v19 activation is buried
-    // and CGovernanceVote::CheckSignature no longer needs to use ::ChainActive()
     LOCK(cs_main);
```

Comment:
@UdjinM6 Well, this one I think we don't have a choice.
`feature_governance.py` fails with `inconsistent lock order for '::cs_main'` on my machine

Reply:
yes, because you didn't really drop `::ChainActive` usage which is what TODO requests as a condition do drop these `cs_main`s. pls see https://github.com/dashpay/dash/pull/5622

Reply:
Right, so should I close this PR since 5622 contains those changes?

Reply:
yeah, that's a part of #5622, see https://github.com/dashpay/dash/pull/5622/commits/21ad17ecd3d6976566f036a2037b35602534088f